src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/imageBase64.js

```diff
@@ -0,0 +1,45 @@
+class ImageBase64Visualization extends Visualization {
+    static inputType = 'Any'
```

Comment:
FYI once I merge my current work on #837 the inputType should be full path for `Image` and `String`.